common/beerocks/bwl/dummy/mon_wlan_hal_dummy.cpp

```diff
@@ -66,9 +66,10 @@ static mon_wlan_hal_dummy::Data dummy_to_bwl_data(const std::string &opcode)
 /////////////////////////////// Implementation ///////////////////////////////
 //////////////////////////////////////////////////////////////////////////////
 
-mon_wlan_hal_dummy::mon_wlan_hal_dummy(const std::string &iface_name, hal_event_cb_t callback)
+mon_wlan_hal_dummy::mon_wlan_hal_dummy(const std::string &iface_name, hal_event_cb_t callback,
+                                       const bwl::hal_conf_t &hal_conf)
     : base_wlan_hal(bwl::HALType::Monitor, iface_name, IfaceType::Intel, callback),
```

Comment:
`base_wlan_hal ctor` also accept `hal_conf` as one of arguments:
```
    base_wlan_hal(HALType type, std::string iface_name, IfaceType iface_type,
                  hal_event_cb_t callback, hal_conf_t hal_conf = {});
```
So pass `hal_conf`into `base_wlan_hal` constructor too

Reply:
Fixed